CONTRIBUTING.md

```diff
@@ -1,29 +1,19 @@
-# REALLY REALLY OUT OF DATE. TODO: update this.
+# Contributing to Chef Cookbooks
 
-
-# Contributing to Chef Software Cookbooks
-
-We are glad you want to contribute to Chef Software Cookbooks! The first
-step is the desire to improve the project.
-
-You can find the answers to additional frequently asked questions
-[on the wiki](http://wiki.chef.io/display/chef/How+to+Contribute).
-
-You can find additional information about
-[contributing to cookbooks](http://wiki.chef.io/display/chef/How+to+Contribute+to+Chef Software+Cookbooks)
-on the wiki as well.
+We are glad you want to contribute to Chef Cookbooks! The first
+step is the desire to improve the project. If you're new to the Chef
+community, please read
+[How to become a contributor](https://supermarket.chef.io/become-a-contributor)
+on the Supermarket website for more information.
 
 ## Quick-contribute
 
-* Create an account on our [bug tracker](http://tickets.chef.io)
-* Sign our contributor agreement (CLA)
-[ online](https://secure.echosign.com/public/hostedForm?formid=PJIF5694K6L)
-(keep reading if you're contributing on behalf of your employer)
-* Create a ticket for your change on the
-  [bug tracker](http://tickets.chef.io)
-* Link to your patch as a rebased git branch or pull request from the
-  ticket
-* Resolve the ticket as fixed
+* Create an account on the [Supermarket](http://supermarket.chef.io)
+* Sign our contributor agreement (CLA)[online](https://supermarket.chef.io/ccla-signatures/new)
```

Comment:
Being this isn't in the chef workspace, i think _technically_ these steps aren't required.